docs/configuration/gpu-health-monitor.md

```
@@ -130,6 +130,28 @@ dcgm:
 
+## DCGM Health Check Incident Suppression
+
+Drops DCGM health check incidents matching specific error codes before they generate a health event, so they are never persisted or acted on. Useful for high-frequency, non-actionable flaps (e.g. normal power-cap boost-clock behavior).
+
+```yaml
+gpu-health-monitor:
+  dcgmHealthCheck:
+    suppressedErrorCodes: []
+```
+
+#### suppressedErrorCodes
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)